# data type correct actual value doesn't make sense


# num = int("abc") -> ValueError: invalid 
# literal for int() with base 10: 'abc'